specification/redisenterprise/resource-manager/readme.md

```diff
@@ -26,10 +26,19 @@ These are the global settings for the RedisEnterprise API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2025-04-01
+tag: package-preview-2025-05-01
 ```
 
 
+### Tag: package-preview-2025-05-01-preview
+
+These settings apply only when `--tag=package-preview-2025-05-01-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2025-05-01-preview'
```

Comment:
please check the tag which is not same with line 29

Reply:
Strange that this wasn't generated correctly by the tool, let me update this.